packages/core/src/Stores/ui-store.js

```diff
@@ -399,6 +407,46 @@ export default class UIStore extends BaseStore {
         this.should_show_cancel = value;
     }
 
+    @action.bound
+    setShouldShowRiskToleranceWarningModal(value) {
+        this.should_show_risk_tolerance_warning_modal = value;
+    }
+
+    @action.bound
+    setIsTradingAssessmentForExistingUserEnabled(value) {
+        this.is_trading_assessment_for_existing_user_enabled = value;
+    }
+
+    @action.bound
+    setIsTradingAssessmentForNewUserEnabled(value) {
+        this.is_trading_assessment_for_new_user_enabled = value;
+    }
+
+    @action.bound
+    setShouldShowAppropriatenessTestWarningModal(value) {
+        this.should_show_appropriateness_test_warning_modal = value;
+    }
+
+    @action.bound
+    setShouldShowAssessmentCompleteModal(value) {
+        this.should_show_assessment_complete_modal = value;
+    }
+
+    @action.bound
+    setShouldShowCooldownWarningModal(value) {
+        this.should_show_cooldown_warning_modal = value;
+    }
+
+    @action.bound
+    setShouldShowTradingAssessmentModal(value) {
+        this.should_show_trading_assessment_modal = value;
+    }
+
+    @action.bound
+    setShouldShowTradeAssessmentForm(value) {
+        this.should_show_trade_assessment_form = value;
+    }
```

Comment:
Can this not be in stores? We only put stuff in stores if it is affecting multiple components